helm/volsync/Chart.yaml

```diff
@@ -21,22 +21,8 @@ annotations:  # https://artifacthub.io/docs/topics/annotations/helm/
   # Changelog for current chart & app version
   # Kinds: added, changed, deprecated, removed, fixed, and security
   artifacthub.io/changes: |
-    - kind: added
-      description: Users can manually label destination Snapshot objects with volsync.backube/do-not-delete to prevent VolSync from deleting them. This provides a way for users to avoid having a Snapshot deleted while they are trying to use it. Users are then responsible for deleting the Snapshot.
-    - kind: added
-      description: Publish Kubernetes Events to help troubleshooting
-    - kind: added
-      description: New Syncthing-based mover for live data replication
-    - kind: changed
-      description: Operator-SDK upgraded to 1.22.0
-    - kind: changed
-      description: Rclone upgraded to 1.59.0
-    - kind: changed
-      description: Restic upgraded to 0.13.1
-    - kind: changed
-      description: Syncthing upgraded to 1.20.1
```

Comment:
Are these (lines 24-37) intentionally removed?  Not sure how it works when updating the changes section without updating the charts version?

Reply:
I pulled them out since the contents were from the last release. ArtifactHub extracts the info to show here: https://artifacthub.io/packages/helm/backube-helm-charts/volsync?modal=changelog
So you replace it all w/ each release.
We'll need to set the version/appVersion prior to the next release.